spec/lib/deployment_deployers_config_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rspec'
 require 'tmpdir'
-require_relative '../../lib/deployment_deployers_config'
-require_relative '../../lib/directory_initializer'
+require 'deployment_deployers_config'
```

Comment:
why we don't need the 'require_relative' ?

Reply:
Regarding `concourse/tasks/[*.sh|*.rb]` and `scripts`, I could not find any occurrence that would be relevant for this ticket.

Reply:
For instance, bosh password (https://github.com/orange-cloudfoundry/cf-ops-automation/blob/87a70ad74d2d92dc06e8242c6e4a0d500145ea1c/concourse/pipelines/template/depls-pipeline.yml.erb#L398) should / may be escaped. We should also check git-credentials.

Reply:
I escaped it but those comments are based on an obsolete version of the code. It's my fault for mixing two threads :)

As for the git passwords, I quoted them but it is not extremely useful since we get problems only with pipeline manifests we're generating ourselves and none of them are using git passwords.